pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -94,7 +94,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, dm *v1alpha1.DomainMappi
 
 	// Reconcile the Ingress resource corresponding to the requested Mapping.
 	logger.Debugf("Mapping %s to ref %s/%s (host: %q, svc: %q)", url, dm.Spec.Ref.Namespace, dm.Spec.Ref.Name, targetHost, targetBackendSvc)
-	desired := resources.MakeIngress(dm, targetBackendSvc, targetHost, ingressClass)
+	desired := resources.MakeIngress(dm, targetBackendSvc, targetHost, ingressClass, []netv1alpha1.IngressTLS{})
```

Comment:
`nil`?

Reply:
Same elsewhere?

Reply:
I will need to do `nil /* tls */` which was why I switched to this to spell out the type.